src/utils/client.ts

```diff
@@ -462,8 +468,11 @@ export default class Client {
       method: 'POST',
       body: JSON.stringify(data)
     };
-    if (promise) return promise.then(() => this._fetch(url, options));
-    else return this._fetch(url, options);
+    // no events for draft
+    const doRegister = () =>
```

Comment:
promise = promise || Promise.resolve();
return promise.then(() => this.draft && this._fetch(...